### ISSUE TITLE
[4.0] System Dashboard accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#25989]
![image](https://user-images.githubusercontent.com/1296369/63575998-3e698080-c583-11e9-831e-ec19b2040640.png)

The screen reader announces "Warnings1" this might cause issues - I _think_ it should be "warnings 1 warning" where the second warning is sr-only

Would be good to get another opinion on that before making a change

The screen reader doesn't announce anything on the database check to indicate everything is ok
